generators/app/index.ts

```diff
@@ -152,38 +165,40 @@ module.exports = yeoman.Base.extend({
         });
 
         this.fs.writeJSON(path.join(this.destinationRoot(), 'package.json'), pkg);
-    },
+        this.fs.move(path.join(this.destinationRoot(), 'dockerignore'), path.join(this.destinationRoot(), '.dockerignore'));
+    }
 
-    _writingVSTSTask: function () {
+    private _writingVSTSTask() {
         this.log(yosay('A new task to make a great platform even better'));
         this._writeSharedVSTSContent();
         this.sourceRoot(vstsRoot);
-        var context = this._buildVSTSContext();
+        const context = this._buildVSTSContext();
+
+        this.fs.copyTpl(this.sourceRoot() + '/**/*', this.destinationRoot(), context);
 
-        this.fs.copyTpl(glob.sync(this.sourceRoot() + '/**/*', { dot: true }), this.destinationRoot(), context);
+        const pkg = this.fs.readJSON(path.join(this.destinationRoot(), 'package.json'), {});
 
-        var pkg = this.fs.readJSON(path.join(this.destinationRoot(), 'package.json'), {});
         extend(pkg, {
             dependencies: {
                 'request': '^2.79.0',
                 'vsts-task-lib': '^1.1.0'
             },
             devDependencies: {
-                '@types/request': '^0.0.36',
+                '@types/request': '^0.0.36'
             }
         });
 
         this.fs.writeJSON(path.join(this.destinationRoot(), 'package.json'), pkg);
-    },
+    }
 
-    _writingChatbotTask: function () {
+    private _writingChatbotTask() {
         this.log(yosay('Engage users in new channels with a Chatbot'));
         this.sourceRoot(chatbotRoot);
-        var context = this._buildExpressContext();
+        const context = this._buildExpressContext();
 
-        this.fs.copyTpl(glob.sync(this.sourceRoot() + '/**/*', { dot: true }), this.destinationRoot(), context);
```

Comment:
weird. either there is an issue with the type def or copyTpl is smart enough to handle an array. as long as we've got the updates for all the other 'dot' files then all good